src/common/Throttle.h

```diff
@@ -330,4 +332,71 @@ class OrderedThrottle {
   uint32_t waiters = 0;
 };
 
+
+class Bucket {
```

Comment:
Yes, we can put it in TokenBucketThrottle. The reason I put it outside is that I was thinking is there a possibility we need to implement another LeakyBucketThrottle which will use the Bucket as well? But that's just a possibility in the future, and we can move it outside on that day. I will put Bucket into TokenBucketThrottle now.